Adyen/Integration/01_ManageAdyenSettings.md

```diff
@@ -0,0 +1,282 @@
+- At asynchronous processing, a message is not processed directly so that a lot of traffic can be handled. The message is first accepted and roughly checked for plausibility. It is then written to a message queue. After that, the message queue is processed periodically and can be run with parallel jobs. 
+
+
+#### Prerequisites
+
+- You have a valid user account in Adyen&trade;.
+- You are assigned to the required roles in Adyen&trade;.
+
+#### Procedure
+
+*Adyen&trade; Backend > Developers > Webhooks*
+
+![Webhook configuration in Adyen](../../Assets/Screenshots/Adyen/Integration/AdyenWebhooks.png "[Webhook configuration in Adyen]")
+
+1. Click on your standard webhook.
+
+2. Expand the *Server configuration* setting in the *General* section.
+
+    ![Define server configuration](../../Assets/Screenshots/Adyen/Integration/AdyenChangeWebhookServerConguration.png "[Define server configuration]").
+
+3. Configure the server URL as follows:   
+    - If you want to process the messages asynchronously, add an **Asynch** to the address. For example: `https://customeraccount.dev.actindo.com/Actindo.Extensions.Actindo.Adyen.Notification.notificationAsync`.
+
+    - If you want to process the messages synchronously, add a **2** to the address. For example: `https://customeraccount.actindo.com/Actindo.Extensions.Actindo.Adyen.Notification.notification2`.
+
+4. Click the [Apply] button.   
+    The synchronous or asynchronous processing has been defined.
+
+5. Click the [Save changes] button in the bottom right corner.
+
+
+
+## Create web service API credentials
+
+Create the web service API credentials to send messages from the *Payments* module to the Adyen&trade; backend. For detailed information, see [API credentials](https://docs.adyen.com/development-resources/api-credentials/ "https://docs.adyen.com/development-resources/api-credentials/") in the Adyen&trade; documentation. 
+
+When you switch to a live environment, you must repeat this procedure in your live customer area.
+
+#### Prerequisites
+
+- You have a valid user account in Adyen&trade;.
+- You are assigned to the required user roles in Adyen&trade;.
+- You are logged in to your Actindo instance under *Payments > Settings > Select connection > Credentials tab*. For detailed information, see [Configure Adyen connection](02_ManageAdyenConnection.md#configure-adyen-connection).
+
+#### Procedure
+
+*Adyen&trade; Backend > Developers > API credentials*
+
+![API credentials](../../Assets/Screenshots/Adyen/Integration/AdyenAPICredentialspng.png "[API credentials]")
+
+1. Click the *Create new credentials* button.   
+   The *Create API credential* window is displayed. The *Web service user* credential type is selected by default. The username was automatically created by the system.
```

Comment:
Vergangenheit da finde ich seltsam... Vorschlag: The username **is** automatically created by the system.